ADAL/src/cache/ADTokenCacheItem.m

```diff
@@ -53,7 +53,7 @@ - (NSUInteger)hash
 
 - (void)calculateHash
 {
-    _hash = [[NSString stringWithFormat:@"%@%@%@%@", _resource, _authority, _clientId, _userInformation.userId] hash];
+    _hash = [[NSString stringWithFormat:@"%@%@%@%@%@", _resource, _authority, _clientId, _userInformation.userId, _applicationIdentifier] hash];
```

Comment:
do we need the enrollment id to calculate the hash?

Reply:
Nope, because enrollmentId is mutable and shouldn't be used as part of the uniqueness check.

Reply:
One more question, why is the `MSIDCredentialCacheItem` hash is enrollment-id-relevant? :)
https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/486/files#diff-1a4dd1c081db0ef0e257e28cdf78a6f3

Reply:
Because it's a bug :) and it shouldn't be